src/banking_stage.rs

```diff
@@ -92,26 +119,28 @@ impl BankingStage {
             }).collect()
     }
 
-    fn tick(poh_service: &PohService, entry_sender: &Sender<Vec<Entry>>) -> Result<()> {
-        let poh = poh_service.tick();
-        let entry = Entry {
-            num_hashes: poh.num_hashes,
-            id: poh.id,
-            transactions: vec![],
-        };
-        entry_sender.send(vec![entry])?;
-        Ok(())
+    fn tick_producer(poh: PohRecorder, config: Config) -> Result<()> {
```

Comment:
you don't want this in the recorder?

Reply:
i think the banking stage should drive this.

Reply:
in that case, why give entry_sender and the bank to the recorder?